insights/specs/default.py

```diff
@@ -348,6 +348,7 @@ def semid(broker):
     ls_var_tmp = simple_command("/bin/ls -ln /var/tmp")
     ls_var_run = simple_command("/bin/ls -lnL /var/run")
     ls_var_spool_postfix_maildrop = simple_command("/bin/ls -ln /var/spool/postfix/maildrop")
+    ls_ocp_cni_openshift_sdn = simple_command("/bin/ls -l /var/lib/cni/networks/openshift-sdn")
```

Comment:
Please add a corresponding `simple_file` spec to `insights/specs/insights_archive.py` as well so that the spec will be accessible from an archive.  You can use the command `mangle "command spec"` to determine the file path to use for `simple_file`.  Also move this up the list so that the variable `ls_ocp_cni_opehshift_sdn` is in sorted order with the other specs.

Reply:
Please insert the new spec in alphabetical order.